src/specmap/lib/all-of.js

```diff
@@ -35,7 +33,7 @@ export default {
     originalDefinitionObj = { ...originalDefinitionObj };
 
     // when we've lost sight, interrupt prematurely
-    if (isEmpty(originalDefinitionObj)) {
+    if (!Object.keys(originalDefinitionObj).length) {
```

Comment:
Nitpick: can we be more explicit here?

```suggestion
    if (Object.keys(originalDefinitionObj).length === 0) {
```

Reply:
done